### PR TITLE
Adding NavBar

### PR DESCRIPTION
**Pull request by Icon1cc**
Created the NavBar as per figma UI. Please review it. 

NavBar SS

### CODE REVIEW

File: src/components/Sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { FiUsers, FiLogOut, FiFileText } from 'react-icons/fi';
+
+const Sidebar = () => {
+    return (
+        <div 
+            className="w-16 h-screen flex flex-col justify-between py-4"
+            style={{ backgroundColor: '#002D7E' }}
```
Comment (DoStini) [inline src/components/Sidebar.tsx:7]: Don't use style to stle background colors, opt to tailwind CSS colors. You can see in @Satvik26 login's page how to use our pallete colors

File: src/components/Sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        >
+            <div>
+                {/* Square with rounded edges containing the "K" */}
+                <div className="flex justify-center mt-4 mb-8">
+                    <div className="bg-[#DEF2FF] rounded-lg w-12 h-12 flex items-center justify-center">
```
Comment (DoStini) [inline src/components/Sidebar.tsx:12]: create predefined colors in the tailwind theme so we can use bg-primary or pg-secondary, etc

File: src/components/Sidebar.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                        <span className="text-[#0099FF] text-3xl font-bold">K</span>
```
Comment (DoStini) [inline src/components/Sidebar.tsx:13]: same here

File: src/components/Sidebar.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                    </div>
+                </div>
+                
+                {/* Profile icon */}
+                <div className="mb-6 flex justify-center">
+                    <button
+                        className="bg-[#003EAE] p-0 rounded-lg flex items-center justify-center"
```
Comment (DoStini) [inline src/components/Sidebar.tsx:20]: same here

File: src/components/Sidebar.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+                        style={{ width: '3rem', height: '3rem' }}
+                    >
+                        <FiUsers size={30} style={{ color: 'white' }} />
+                    </button>
+                </div>
+
+                {/* Sheets icon */}
+                <div className="mb-6 flex justify-center">
+                    <button
+                        className="bg-[#003EAE] p-0 rounded-lg flex items-center justify-center"
```
Comment (DoStini) [inline src/components/Sidebar.tsx:31]: sam here

**Review by DoStini — CHANGES_REQUESTED** [review]

**Comment by Icon1cc** [comment]
Updated the code as per Tailwind CSS as request. Please let me know if any further change is required. Thanks

File: src/components/Sidebar.tsx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+                </div>
+
+                {/* Sheets icon */}
+                <div className="mb-6 flex justify-center">
+                    <button
+                        className="bg-blue-800 p-0 rounded-lg flex items-center justify-center w-12 h-12"
+                        title="Exercise List"
+                    >
+                        <FiFileText size={30} className="text-white" />
+                    </button>
```
Comment (DoStini) [inline src/components/Sidebar.tsx:32]: After adding the new routes, use this component instead of a button https://reactrouter.com/en/main/components/link

**Review by DoStini — CHANGES_REQUESTED** [review]

File: src/components/Sidebar.tsx [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+import { ImProfile } from 'react-icons/im';
+
+const Sidebar = () => {
+    return (
+        <div className="w-16 h-screen flex flex-col justify-between py-4 bg-blue-900">
+            <div>
+                {/* Square with rounded edges containing the "K" */}
+                <div className="flex justify-center mt-4 mb-8">
+                    <div className="bg-blue-100 rounded-lg w-12 h-12 flex items-center justify-center">
+                        <span className="text-blue-500 text-3xl font-bold">K</span>
```
Comment (DoStini) [inline src/components/Sidebar.tsx:11]: Replace the text blue with our own colors

**Review by DoStini — CHANGES_REQUESTED** [review]

**Review by DoStini — APPROVED** [review]

**Pull request merged**